auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/EntitlementService.java

```diff
@@ -613,6 +614,10 @@ public boolean isTargetingSubnetsForEndpointAccessGatewayEnabled(String accountI
         return isEntitlementRegistered(accountId, TARGETING_SUBNETS_FOR_ENDPOINT_ACCESS_GATEWAY);
     }
 
+    public boolean isWiamUsersyncEnabled(String accountId) {
```

Comment:
I would rename this to `isWiamUsersyncRoutingEnabled` to better match the entitlement name